http/src/main/resources/reference.conf

```diff
@@ -807,6 +807,7 @@ refererConfig {
   # list of valid hosts and enabled flag is environment specific so it's overriden in firecloud develop
   validHosts = []
   enabled = false // false for dev and QA environments, true everywhere else
+  originStrict = false // true only if "*" in validHosts is to be ignored by origin validation
```

Comment:
Can you link the sister PR in terra helmfile that does this? it is related to Rob's comment above

Reply:
There's no sister PR because this config is really only needed in debugging scenarios, in environments (BEEs) where the "*" in validHosts is required for functionality. So this defaults to false and can be optionally set to true to tweak behavior (when you want referers permissive but origins strict). Once we're happy with this PR, we could optionally remove this setting entirely.